EIPS/eip-5115.md

```diff
@@ -0,0 +1,397 @@
+- **MUST** be emitted when rewards are claimed from the SCY contract via `claimRewards` method.
+
+```solidity
+event ExchangeRateUpdated(uint256 oldExchangeRate, uint256 newExchangeRate);
+```
+
+The `exchangeRateCurrent` is updated to the latest exchange rate.
+
+- **MUST** be emitted when the exchange rate is updated in the SCY contract via `exchangeRateCurrent` method.
+
+**"SCY" Word Choice:**
+
+"SCY" (pronunciation: */sʌɪ/*), an abbreviation of Super Composable Yield, was found to be appropriate to describe a broad universe of composable yield-bearing digital assets.
+
+## Rationale
+
+This EIP is designed for flexibility, aiming to accommodate as many yield generating mechanisms as possible. Particularly, this standard aims to be generalized enough that it supports the following uses cases and more:
+
+- Money market supply positions
+    - Lending DAI in Compound, getting DAI interests and COMP rewards
+    - Lending ETH in BenQi, getting ETH interests and QI + AVAX rewards
+    - Lending USDC in Aave, getting USDC interests and stkAAVE rewards
+- AMM liquidity provision
+    - Provide ETH + USDC to ETHUSDC pool in SushiSwap, getting swap fees in more ETH+USDC
+    - Provide ETH + USDC to ETHUSDC pool in SushiSwap and stake it in Sushi Onsen, getting swap fees and SUSHI rewards
+    - Provide USDC+DAI+USDT to 3crv pool and stake it in Convex, getting 3crv swap fees and CRV + CVX rewards
+- Vault positions
+    - Provide ETH into Yearn ERC-4626 vault, where the vault accrues yield from Yearn’s ETH strategy
+    - Provide DAI into Harvest and staking it, getting DAI interests and FARM rewards
+- Liquid staking positions
+    - Holding stETH (in Lido), getting yields in more stETH
+- Liquidity mining programs
+    - Provide USDC in Stargate, getting STG rewards
+    - Provide FEI in Tokemak, getting TOKE rewards
+    - Provide LOOKS in LooksRare, getting LOOKS yield and WETH rewards
+- Rebasing tokens
+    - Stake OHM into sOHM/gOHM, getting OHM rebase yield
+    - Stake BTRFLY into xBTRFLY, getting BTRFLY rebase yield
+
+The EIP hopes to minimize, if not possibly eliminate, the use of customized adapters in order to interact with many different forms of yield-generating token mechanisms.
+
+ERC-20 is enforced because implementation details such as transfer, token approvals, and balance calculation directly carry over to the SCY tokens. This standardization makes the SCY tokens immediately compatible with all ERC-20 use cases.
+
+## Backwards Compatibility
+
+This EIP is fully backwards compatible as its implementation extends the functionality of [ERC-20](./eip-20.md), however the optional metadata extensions, namely `name`, `decimals`, and `symbol` semantics MUST be implemented for all SCY token implementations.
+
+## Security Considerations
+
+Malicious implementations which conform to the interface can put users at risk. It is recommended that all integrators review the implementation to avoid possible exploits and users losing funds.
```

Comment:
Integrators in this context are? Wallet devs?

Reply:
Yes. Made this clearer by specifying wallets, aggregators, and other smart contract protocols as integrators.